restaurant/src/app/welcome-page/welcome-page.component.ts

```diff
@@ -12,4 +12,8 @@ export class WelcomePageComponent {
   toMenu() {
     this.router.navigate(['menu']);
   }
+
+  goToAboutPage() {
+    this.router.navigate(['about']);
```

Comment:
Vezbaj ovde malo funkcije. Napravi funkciju koja ce primati route string kao properti i posle je pozivaj za toMenu and goToAboutPage